hi-ml-histopathology/testhisto/testhisto/mocks/slides_generator.py

```diff
@@ -188,12 +186,14 @@ def _create_multi_resolution_wsi(self, mock_image: np.ndarray) -> List[np.ndarra
     def generate_mock_histo_data(self) -> None:
         """Create mock wsi and save them as tiff files"""
         iterator = iter(self.dataloader) if self.dataloader else None
-        os.makedirs(self.tmp_path / "train_images", exist_ok=True)
-        os.makedirs(self.tmp_path / "dump_tiles", exist_ok=True)
+        os.makedirs(self.dest_data_path / "train_images", exist_ok=True)
```

Comment:
FYI, Path has it's own inbuilt method 'mkdir' that can do this same thing: 
`(self.dest_data_path / "train_images").mkdir(parents=True, exist_ok=True)`